pageserver/src/tenant/storage_layer/split_writer.rs

```diff
@@ -80,18 +80,13 @@ impl SplitImageLayerWriter {
         })
     }
 
-    pub async fn put_image_with_discard_fn<D, F>(
+    pub async fn put_image(
         &mut self,
         key: Key,
         img: Bytes,
-        tline: &Arc<Timeline>,
+        #[allow(unused)] tline: &Arc<Timeline>,
```

Comment:
Will we need it again?

Reply:
there is a recent change https://github.com/neondatabase/neon/commit/3454ef7507000f05707418324dad392b366f8de3 that completely removes ctx reference, very likely that I can remove this forever.